src/components/App.jsx

```diff
@@ -25,7 +26,10 @@ const App = props => {
       </Sidebar>
 
       <Main>
-        <Content>{props.children}</Content>
+        <Content>
+          {flag('bi-alert') && <KonnectorUpdateInfo />}
```

Comment:
How does it work on the phone when you are not connected?

Reply:
You want to say when we are offline?